prov/efa/src/efa.h

```diff
@@ -132,6 +132,8 @@ struct efa_ah {
 struct efa_conn {
 	struct efa_ah		ah;
 	struct efa_ep_addr	ep_addr;
+	fi_addr_t		fi_addr;
```

Comment:
Typo in commit prov/efa: add fi_addr and rdm_peer into efa_conn
```
simpilication -> simplification
```

Reply:
Could not find it?